Libraries/LibWeb/HTML/RenderingThread.cpp

```diff
@@ -26,8 +26,8 @@ RenderingThread::RenderingThread()
 
 RenderingThread::~RenderingThread()
 {
-    m_exit = true;
-    m_rendering_task_ready_wake_condition.signal();
```

Comment:
This also causes us to lock the mutex before setting m_exit to true and calling condition.signal() from the destructor, but that's probably fine and better anyway.